lib/npg_pipeline/base/options.pm

```diff
@@ -63,6 +63,20 @@ sub _default_to_local {
   return $self->local;
 }
 
+=head2 no_s3_archival
```

Comment:
I foresee a naming problem here as:
- we may provide data products in a temporary manner via our S3 system #330 (which this attribute could be confused with)
- the external customer archive (typically to be used as an alternative to iRODS archival) may not always be S3 - we may be called upon to sync using sftp instead of to an AWS S3 (I'm imagining a per-study URL defining this) - and this attribute would still be the relevant one

Reply:
My thinking here was that for any study we have the option of `iRODS`, `S3`, `SFTP` or somehting else (one, or more of these) - the specific URL(s) being defined by the study. The existing pattern (wrt iRODS) for deciding whether to use a type of storage seems to be a pipeline level switch, so I followed that example.

Yes, this attribute would still be the relevant one - that's what I intended. (I haven't added an `SFTP` option because we don't know whether that will happen for sure.) i.e. if you turn on `iRODS` and `S3` for a study, it goes to both, using the appropriate URLs for that study. If `S3` is temporary, then later you turn it off and turn on `SFTP`, or whatever.